### DISCUSSION TITLE
Storing logs for airflow.utils.email.send_email_smtp default backend

### DISCUSSION THREAD

**Discussion by thegunner157** [apache/airflow#34202] [General]
Hi, can we somehow check where logs are stored for default backend: airflow.utils.email.send_email_smtp? And if not, is it possible to somehow set this up?

**Comment by potiuk** [comment]
I don't think there are any logs generated in those methods. Did you find any `logging.` entry that you are particularly interested in? Or you are really interested in tracking emails sent by Airflow (in the second case you should rather plug-in in the smtp server logs of yours - but that's entirely outside of Airflow.

**Reply by thegunner157** [reply]
I was thinking about second option, as my service account after issue with SLA started to attempt sending emails twice a minute beyond the fact that task was stopped so there was no way to get any logs what was happening

**Reply by potiuk** [reply]
So you should look in the regular SLA logs I guess. There is nothing like "send_email_smtp" logs - there are general logs of the component that uses it and you should look there.
